README.md

```diff
@@ -0,0 +1,18 @@
+# TitanicProbability
+
+## Worum gehts?
+
+Es geht um ein Spring Boot Test Projekt, welches aus Daten zu Passagieren von der Titanic (Name, Alter Geschlecht, Klasse, usw.) aus einer CSV Datei importiert und dann in deiner Postgres Datenbank speichert. 
```

Comment:
Es geht um ein Spring Boot Test Projekt, welches Daten zu Passagieren von der Titanic (Name, Alter Geschlecht, Klasse, usw.) aus einer CSV Datei importiert und dann in einer Postgres Datenbank speichert.